manifest/manifest_shacl.ttl

```diff
@@ -0,0 +1,99 @@
+@prefix manifest: <https://github.com/GAIA-X4PLC-AAD/ontology-management-base/tree/main/manifest/> .
+@prefix gx: <https://registry.lab.gaia-x.eu/development/api/trusted-shape-registry/v1/shapes/jsonld/trustframework#> .
+@prefix sh: <http://www.w3.org/ns/shacl#> .
+@prefix skos: <http://www.w3.org/2004/02/skos/core#> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+
+manifest:ManifestShape a sh:NodeShape ;
+    sh:property             
+        [   sh:description "Description of the content and filestructure of the uploaded asset.zip"@en ;
+            sh:maxCount 1 ;
+            sh:minCount 1 ;
+            sh:node manifest:DataShape ;
+            sh:order 1 ;
+            sh:path manifest:data ];         
+    sh:targetClass manifest:Manifest .
+    
+manifest:DataShape a sh:NodeShape ;
+    sh:property 
+        [   skos:example "link to valuable data, e.g. ./data/hdmap.xodr" ;
+            sh:description "Reference to valuable data/asset as relativePath or link"@en ;
+            sh:message "Validation of data failed!"@en ;
+            sh:minCount 1 ;
+            sh:name "assetData"@en ;
+            sh:node manifest:LinkShape ;
+            sh:order 1 ;
+            sh:path manifest:assetData
+        ],
+        [   skos:example "license type" ;
+            sh:in ("openSource" "providerSpecific") ;
+            sh:message "Validation of license type failed!"@en ;
```

Comment:
I am missing a `description` field for this property

Reply:
done